src/wattsi.pas

```diff
@@ -681,11 +686,61 @@    TCrossReferences = record
                            Scratch.Append($002E);
                         Scratch.Append(IntToStr(CurrentSectionNumber[CurrentHeadingRank]));
                      end;
+                     if (Variant = vDEV) then
+                     begin
+                        ID := Element.GetAttribute('id').AsString;
+                        SectionNumber := Scratch.AsString;
+                        HeadingText := DelSpace1(StringReplace(StringReplace(Element.TextContent.AsString, #$0A, ' ', [rfReplaceAll]), #$22, '\' + #$22, [rfReplaceAll]));
```

Comment:
Maybe it should even be factored out into a function as I see it's repeated a bit below.